docs/githubteacher-bio.md

```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Hometown:** San Francisco
```

Comment:
I think we should be more specific.  There are a lot of San Franciscos in the :earth_africa:.